pkg/clusterversion/clusterversion.go

```diff
@@ -3,17 +3,25 @@ package clusterversion
 import (
 	"context"
 	"fmt"
+	"time"
 
+	"github.com/Masterminds/semver/v3"
 	"github.com/golang/glog"
 	"github.com/openshift-kni/eco-goinfra/pkg/clients"
 	"github.com/openshift-kni/eco-goinfra/pkg/msg"
 	v1 "github.com/openshift/api/config/v1"
 	k8serrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	goclient "sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 const (
-	clusterVersionName = "version"
+	clusterVersionName                        = "version"
+	fiveScds                    time.Duration = 5 * time.Second
+	isTrue                                    = "True"
+	updateHistoryPartialState                 = "Partial"
```

Comment:
This variable used only once. Please remove this var and use string directly in the required place.

Reply:
Done, thanks